epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -142,6 +155,43 @@ private boolean isFieldPackagePrivate(Element attribute) {
   }
 
   /**
+   * Checks if the given private field has getter and setter for access to it
+   */
+  private void findGetterAndSetterForPrivateField(ErrorLogger errorLogger) {
+    for (Element element : classElement.getEnclosedElements()) {
+      if (element.getKind() == ElementKind.METHOD) {
+        ExecutableElement method = (ExecutableElement) element;
+        String methodName = method.getSimpleName().toString();
+        // check if it is a valid getter
+        if (methodName.equalsIgnoreCase(String.format("get%s", name))
+            || methodName.equalsIgnoreCase(String.format("is%s", name))
```

Comment:
ah your tests are failing because your order of operations for the logic here is messed up. You need `( )` around your `||` operation
```
(methodName.equalsIgnoreCase(String.format("get%s", name))
            || methodName.equalsIgnoreCase(String.format("is%s", name)))
```
Tricky!

Btw I recently learned how to debug annotation processing and it is very helpful. I used it to figure this out https://blog.xmartlabs.com/2016/03/28/Debugging-an-Annotator-Processor-in-your-project/